src/components/ui/TextArea.tsx

```diff
@@ -56,8 +56,7 @@ const textBox = classnames(
   backgroundColor('bg-primary-dark'),
   resize('resize-none'),
   width('w-full'),
-  transitionProperty('transition-colors'),
-  outlineStyle('outline-none', 'focus:outline-none')
+  transitionProperty('transition-colors')
```

Comment:
No worries, other outlines do work fine because they're class-styles

https://user-images.githubusercontent.com/49658988/193861047-75727ad9-14e9-4391-a6a5-30d6694f1913.mp4